.rubocop.yml

```diff
@@ -13,4 +13,8 @@ Metrics/PerceivedComplexity:
 
 Metrics/CyclomaticComplexity:
   Exclude:
-    - 'bin/bundle'
\ No newline at end of file
+    - 'bin/bundle'
```

Comment:
This exclude can be eliminating by telling rubocop to skip certain directories and files. Here's a good example:

https://github.com/pulibrary/figgy/blob/main/.rubocop.yml#L7-L15

You won't need the `vendor` and `config/deploy.rb` paths, but the rest of them are good to skip in a rails project. I would add that and then run rubocop again to make sure everything is still passing.

Reply:
Just remove the unneeded lines 18-21 in your .rubocop.yml file and this comment is resolved. https://github.com/pulibrary/book-trade-xander/blob/9c5da53a7215322cab831a2453530d418a4e38cf/.rubocop.yml#L18-L21